doc/services/portability/posix/conformance/index.rst

```diff
@@ -80,7 +80,7 @@ POSIX System Interfaces
 
     _POSIX_ADVISORY_INFO, -1,
     _POSIX_CPUTIME, 200809L, :kconfig:option:`CONFIG_POSIX_CLOCK`
-    _POSIX_FSYNC, -1,
+    _POSIX_FSYNC, 200809L, :kconfig:option:`CONFIG_FDTABLE`
```

Comment:
Ideally, this would be `CONFIG_POSIX_FSYNC`, but the following should be slightly more accurate because that option gates fs.c in CMakeLists.txt
```suggestion
    _POSIX_FSYNC, 200809L, :kconfig:option:`CONFIG_POSIX_FS`
```

Reply:
This is also mentioned down at the bottom of this doc under XSI System Interfaces